t/ust.t

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Fatal;
+use Test::MockModule;
+use Net::Async::Blockchain::Transaction;
+use Net::Async::Blockchain::Omni;
+use Net::Async::Blockchain::Client::RPC::Omni;
+
+my $loop          = IO::Async::Loop->new();
+my $currency_code = 'UST';
+
+my $mocked_omni     = Test::MockModule->new('Net::Async::Blockchain::Omni');
+my $mocked_rpc_omni = Test::MockModule->new('Net::Async::Blockchain::Client::RPC::Omni');
+
+subtest "Omni Send " => sub {
+    $loop->add(my $subscription_client = Net::Async::Blockchain::Omni->new(currency_symbol => $currency_code));
+    my $from = 'mgVxUb5mYJkfoo4w2JBVBcWaP5bqtLroTr';
+    my $to = 'mvaK72Z5EVYJtptWrD2nCSZd1nqEx5B469';
+    my $hash = '19e5bda81d0a588cbf46072fb6da01685a51082d4b94428e09f491ad8a111bc7';
+    my $blockhash = '00000000000b031f58f9761fd1e0219ef5178481f274b4f54d30c5142b97571f';
+    
+    my $expected_transaction = Net::Async::Blockchain::Transaction->new(
+        currency     => $currency_code,
+        hash         => $hash,
+        block        => '1666276',
+        from         => $from,
+        to           => $to,
+        amount       => Math::BigFloat->new(21),
```

Comment:
You don't seem to `use` Math::BigFloat`  before using it.

Reply:
We are declaring it here https://github.com/binary-com/perl-Net-Async-Blockchain/blob/master/lib/Net/Async/Blockchain/Omni.pm#L42. So it's usable without declaring it...